src/Entity/Skill.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\SkillRepository")
+ * @ORM\Table(
+ *   name="skills",
+ *   uniqueConstraints={
+ *     @ORM\UniqueConstraint(name="skill_name_unique", columns="name")
+ *   }
+ * )
+ *
+ * @UniqueEntity("name")
+ */
+class Skill
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(unique=true)
+     *
+     * @Assert\NotBlank
+     * @Assert\Length(min=2, max=200)
+     */
+    private $name = '';
```

Comment:
I don't see any slug property anymore, the unique constraint is not enough if the value is not normalised at some point, maybe using `mb_strtolower`?

Also the initialization is pointless due to the constructor.